crates/detect-targets/src/detect/windows.rs

```diff
@@ -1,11 +1,20 @@
 use crate::TARGET;
 use guess_host_triple::guess_host_triple;
 
-pub(super) fn detect_alternative_targets(target: &str) -> Option<String> {
+pub(super) fn detect_alternative_targets(target: &str) -> impl Iterator<Item = String> {
     let (prefix, abi) = target.rsplit_once('-')?;
 
+    // AFAIK only windows 10/11 supports arm and it requires > 4G of ram,
+    // which makes running it on armv7 moot.
+    let is_arm = prefix.starts_with("aarch64");
```

Comment:
@passcod I'm not sure whether windows supports running unmodified x86 on other arm targets.

I've found `thumbv7a-pc-windows-msvc` which seems to be arm but not sure about that.

Also, I've found several `uwp` variants, shall we also support that?
If so, then I will open another PR after this one is merged.